go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/coredns/corefile-migration v1.0.16
 	github.com/davecgh/go-spew v1.1.1
-	github.com/docker/distribution v2.8.0+incompatible
+	github.com/docker/distribution v2.8.1+incompatible
```

Comment:
Sure, I included that fix because without it, I wasn't able to compile the code. They did something that broke the `go.mod` sum. I'll submit another PR.